client/me/purchases/manage-purchase/index.jsx

```diff
@@ -173,7 +173,7 @@ const ManagePurchase = React.createClass( {
 			const expiryMoment = this.moment( purchase.expiryMoment );
 			const daysToExpiry = this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' );
 
-			return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s days. ',
+			return this.translate( '%(purchaseName)s will expire and be removed from your site in %(expiry)s days. ',
```

Comment:
This looks fine and better than it is, but could you use [plural form](https://github.com/Automattic/i18n-calypso#translate-method) so that it reads "... will expire and be removed from your site in 1 day."